verifier/src/lib.rs

```diff
@@ -169,7 +169,7 @@ pub fn verify<AIR: Air>(
 /// attests to a correct execution of the computation specified by the provided `air`.
 fn perform_verification<A, E, H>(
     air: A,
-    mut channel: VerifierChannel<E, H>,
+    mut channel: VerifierChannel<E, H, A::Frame<E>, A::AuxFrame<E>>,
```

Comment:
I think the type parameter of `A::AuxFrame` might be not necessarily equal to the one of `A::Frame`

Reply:
Nope, my mistake. From the point of view of the verifier both frames are defined over the same field (I was thinking on the prover where they are defined over different fields)